src/tools/stories.ts

```diff
@@ -175,6 +175,24 @@ The story will be added to the default state for the workflow.
 			async ({ storyPublicId }) => await tools.unassignCurrentUserAsOwner(storyPublicId),
 		);
 
+		server.tool(
+			"create-story-comment",
+			"Create a comment on a story",
+			{
+				storyPublicId: z.number().positive().describe("The public ID of the story"),
+				text: z.string().min(1).describe("The text of the comment"),
+				authorId: z
```

Comment:
I'm wondering if we should leave this out? I'm not sure we want or need to support creating comments as other users. Can you think of a use case? And if we don't need it, it might be better to leave it out to avoid confusing the AI.